oci/archive/oci_transport.go

```diff
@@ -159,8 +159,16 @@ func (t *tempDirOCIRef) deleteTempDir() error {
 }
 
 // createOCIRef creates the oci reference of the image
+// Deprecated: Please use createOCIRefWithSystemContext which will allows you to configure temp directory
+// for big files through SystemContext.BigFilesTemporaryDir
 func createOCIRef(image string) (tempDirOCIRef, error) {
```

Comment:
This is a package-private function, so it can just be modified without maintaining a compatibility shim.